pkg/cmd/info/info.go

```diff
@@ -0,0 +1,63 @@
+package info
+
+import (
+	"context"
+	"errors"
+	"fmt"
+	"kitops/pkg/artifact"
+	"kitops/pkg/lib/constants"
+	"kitops/pkg/lib/repo"
+	"kitops/pkg/output"
+
+	"oras.land/oras-go/v2/errdef"
+)
+
+func getInfo(ctx context.Context, opts *infoOptions) (*artifact.KitFile, error) {
+	if opts.modelRef.Registry == repo.DefaultRegistry {
+		// Local only check
+		return getLocalConfig(ctx, opts)
+	}
+	if opts.checkRemote {
+		// Remote only check
+		return getRemoteConfig(ctx, opts)
+	}
+
+	// Check locally first; if not found check remote
+	manifest, err := getLocalConfig(ctx, opts)
```

Comment:
Should we actually check if the remote modelkit SHA matches the local one? This is for the case where somebody else moves the tag and now the same tag is referring to different versions on local and remote.

Reply:
It's tricky -- we don't necessarily want to be pinging the remote registry if someone intends to work locally, and we can't necessarily show useful information in case the digest associated with a tag has diverged (e.g. did I build a new ModelKit locally or did the remote one update?). We can maybe improve this via timestamps or similar, though.

For now I'm opting for the semantics of docker/podman, where local vs remote tags can diverge and it's on the user to figure out what to do. I think docker goes one step further and only supports inspecting local images -- this may be a better choice for us as well. 

What do you think about simplifying `info` and `inspect` to work local-only by default, with the optional `-r|--remote` flag available to check remotes explicitly?

Reply:
It is a good idea to make remote checking explicit with `-r` flag. We can even print a message when we notice that remote and local are not pointing to the same SHA.

Reply:
Done :+1: